lms/static/js/instructor_dashboard/membership.js

```diff
@@ -7,16 +7,18 @@ imports from other modules.
 wrap in (-> ... apply) to defer evaluation
 such that the value can be defined later than this assignment (file load order).
 */
-
+var Membership;
```

Comment:
Let me check the window object.  I did not want to move it outside, but was unable to see the Object in the test otherwise.